code/__HELPERS/admin.dm

```diff
@@ -55,4 +55,5 @@ params
 
 		if(GLOB.admin_approvals[approval_id] != -1)
 			. = options[GLOB.admin_approvals[approval_id]]
+			log_admin_private("[user] has .resolved admin approval [admin_message]")
```

Comment:
This is already logged, unless im missing something here?
```else if(href_list["adminapproval"])
		var/approval_id = href_list["adminapproval"] // Already text at this point
		if(GLOB.admin_approvals[approval_id] != -1)
			to_chat(usr, "<span class='warning'>That approval has already been answered with '[GLOB.admin_approvals[approval_id]]'</span>")
			return
		GLOB.admin_approvals[approval_id] = href_list["option"]
		log_admin("[key_name(usr)] answered '[href_list["option"]]' to the admin approval ([approval_id]).")
		message_admins("[key_name(usr)] answered '[href_list["option"]]' to the admin approval ([approval_id]).")```

Reply:
oops